README.md

```diff
@@ -18,6 +19,9 @@ roots = []
 comment-token = "#"
 language-servers = ["roc-ls"]
 indent = { tab-width = 2, unit = "  " }
+auto-format = true
+formatter = { command = "roc", args =[ "format", "--stdin", "--stdout"]}
```

Comment:
Are you sure this is needed? Mine is formatting with just the language server configured.

Reply:
That's interesting, leave them in for now, sounds like it could be a bug in the language server.